src/overlays/actors/ovl_En_Ta/z_en_ta.c

```diff
@@ -107,80 +146,83 @@ void EnTa_Init(Actor* thisx, PlayState* play2) {
     Collider_SetCylinder(play, &this->collider, &this->actor, &sCylinderInit);
 
     this->actor.colChkInfo.mass = MASS_IMMOVABLE;
-    this->unk_2E0 = 0;
-    this->unk_2CE = 0;
-    this->unk_2E2 = 0;
+    this->stateFlags = 0;
+    this->rapidBlinks = 0;
+    this->nodOffTimer = 0;
     this->blinkTimer = 20;
-    this->unk_2B0 = func_80B166CC;
+    this->blinkFunc = EnTa_BlinkWaitUntilNext;
     Actor_SetScale(&this->actor, 0.01f);
     this->actor.targetMode = 6;
     this->actor.velocity.y = -4.0f;
     this->actor.minVelocityY = -4.0f;
     this->actor.gravity = -1.0f;
 
     switch (this->actor.params) {
-        case 1:
+        case ENTA_IN_KAKARIKO:
+            // "Exile Talon"
             osSyncPrintf(VT_FGCOL(CYAN) " 追放タロン \n" VT_RST);
-            if (GET_EVENTCHKINF(EVENTCHKINF_6B)) {
+            if (GET_EVENTCHKINF(EVENTCHKINF_TALON_RETURNED_FROM_KAKARIKO)) {
                 Actor_Kill(&this->actor);
             } else if (!LINK_IS_ADULT) {
                 Actor_Kill(&this->actor);
-            } else if (GET_EVENTCHKINF(EVENTCHKINF_6A)) {
-                func_80B13AA0(this, func_80B14CAC, func_80B167C0);
-                this->eyeIndex = 0;
+            } else if (GET_EVENTCHKINF(EVENTCHKINF_TALON_WOKEN_IN_KAKARIKO)) {
+                EnTa_SetupAction(this, EnTa_IdleWokenInKakariko, EnTa_AnimLoopCurrent);
+                this->eyeIndex = TALON_EYE_INDEX_OPEN;
                 Animation_PlayOnce(&this->skelAnime, &gTalonStandAnim);
                 this->currentAnimation = &gTalonStandAnim;
             } else {
-                func_80B13AA0(this, func_80B14754, func_80B167FC);
-                this->eyeIndex = 2;
+                EnTa_SetupAction(this, EnTa_IdleAsleepInKakariko, EnTa_AnimSleeping);
+                this->eyeIndex = TALON_EYE_INDEX_CLOSED;
                 Animation_PlayOnce(&this->skelAnime, &gTalonSleepAnim);
                 this->currentAnimation = &gTalonSleepAnim;
                 this->actor.shape.shadowScale = 54.0f;
             }
             break;
-        case 2:
+        case ENTA_RETURNED_FROM_KAKARIKO:
```

Comment:
It is a little nicer to have an empty line between `break`s and the next case to show they are separate (vs fall-through cases where the new line is ommited).